react/components/molecules/select/select.js

```diff
@@ -0,0 +1,146 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, ViewPropTypes, View } from "react-native";
+import PropTypes from "prop-types";
+import RNPickerSelect from "react-native-picker-select";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles, equal } from "../../../util";
+
+import { Icon } from "../../atoms";
+
+export class Select extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            options: PropTypes.array,
+            value: PropTypes.any,

```

Comment:
How does it behave with empty options?

Reply:
Empty select: <br>
| - | - |
| --- | --- |
| iOS | Android |
| <img width="441" alt="imagem" src="https://user-images.githubusercontent.com/25725586/121040641-f7756b00-c7a9-11eb-8579-9ce2d3d36d84.png"> | ![yoyo](https://user-images.githubusercontent.com/25725586/121041208-72d71c80-c7aa-11eb-8fe7-d1ab19e9628e.gif) |

Reply:
Please confirm that ripe-components-vue has the same behavior

Reply:
Its not the same bahaviour, but porting it is tricky. The problem is in react-native, the only way to do this is to set an option with a `null` value, which triggers a warning. There is a feature request to disable certain options https://github.com/lawnstarter/react-native-picker-select/issues/197, but its not implemented.

| - | - |
| --- | --- |
| components-vue | react-native similar |
| ![select-components-vue](https://user-images.githubusercontent.com/25725586/121171551-89cf4a80-c84e-11eb-891e-e705404187f3.gif) | ![no-items-select](https://user-images.githubusercontent.com/25725586/121171559-8c31a480-c84e-11eb-952b-92da84588570.gif) |